src/program.ts

```diff
@@ -2987,12 +3000,21 @@ export class File extends Element {
   }
 
   /** Creates an imported namespace from this file. */
-  asImportedNamespace(name: string, parent: Element, localIdentifier: IdentifierExpression): Namespace {
+  asImportedNamespace(
```

Comment:
If you think that `aliasNamespaces` is good, we may also rename this one to `asAliasNamespace`.